packages/ra-ui-materialui/src/layout/Confirm.tsx

```diff
@@ -126,7 +128,7 @@ export interface ConfirmProps extends Omit<DialogProps, 'open' | 'onClose'> {
     loading?: boolean;
     onClose: MouseEventHandler;
     onConfirm: MouseEventHandler;
-    title: string;
+    title: React.ReactNode;
```

Comment:
You need to omit `'title'` from `DialogProps` too I believe (cause  `'title'` in `DialogProps`  only supports `string`)

Reply:
`DialogProps` not have `title` prop, because it uses component for it `<DialogTitle />`. 
As well I don't see it in types.

Reply:
Hm, I don't see `title` type there and in Modal types too 🤷🏻‍♂️ 
By the way I going to try omit it and will see if it helps.